brandon_tan/final_submission.ipynb

```diff
@@ -0,0 +1,436 @@
+{
+ "metadata": {
```

Comment:
The notebook is good but would benefit from a body section that describes the various techniques.